sentry-ruby/spec/sentry/configuration_spec.rb

```diff
@@ -154,6 +173,16 @@
     end
   end
 
+  describe "#profiles_sample_rate" do
+    it "returns nil by default" do
```

Comment:
Let's replicate `traces_sample_rate`'s tests here if you agree with my suggestions.